addons/web/static/src/core/model_field_selector/model_field_selector_popover.js

```diff
@@ -23,11 +29,21 @@ class Page {
 
     get path() {
         const previousPath = this.previousPage?.path || "";
-        if (this.selectedName) {
+        let name = this.selectedName;
+        if (this.readProperty) {
+            if (this.selectedField && this.selectedField.type === "properties") {
+                name = `_read_property('${this.selectedName}')`;
```

Comment:
Since they are the same object.
```suggestion
                name = `_read_property('${name}')`;
```